setup.cfg

```diff
@@ -51,6 +51,7 @@ exclude_lines =
 max-line-length = 88
 ignore = E203, E501, W503
 exclude = .git, __pycache__, gcovr/tests/cmake_gtest/build/_deps
+builtins = Options
```

Comment:
This is needed for proper type-defing of "Options" within gcovr/coverage.py. It seems I cannot import this from __main__.py, so I must tell flake that it's a valid class. Interested in other ways to do this properly, that would work with flake8 (or mypy)

Reply:
I think it's better to move the class to utils because it doesn't have a dependency.
This will also remove the error which is shown by Pylance in VSCode.